feature/src/common/frame.h

```diff
@@ -19,20 +19,25 @@
 #ifndef FRAME_H_
 #define FRAME_H_
 
+#include <string.h>
+#include "alloc.h"
+#include "libvmaf.h"
+
 struct data
 {
-    char* format; /* yuv420p, yuv422p, yuv444p, yuv420p10le, yuv422p10le, yuv444p10le */
+    enum VmafPixelFormat format; /* yuv420p, yuv422p, yuv444p, yuv420p10le, yuv422p10le, yuv444p10le */
     int width;
     int height;
     size_t offset;
     FILE *ref_rfile;
     FILE *dis_rfile;
     int num_frames;
+    int use_color;
```

Comment:
why `use_color`?

Reply:
I'm confused by this struct. It's like a copy of the `VmafPicture` struct.

Reply:
use_color was used to determine whether the read_frame/read_vmaf_picture should populate or skip the U and V planes. If we decide we always wanna do that, we can remove this field.

Reply:
This struct is the definition of user_data under the context of read_frame. It does not only have things like width and height as in VmafPicture, it also has the FILE* to ref and dis files.

Reply:
As of the latest commits, use_color has now been removed, and we explicitly ask for a chroma feature, like PSNR using the VmafFeatureModeSetting enum.